pom.xml

```diff
@@ -154,19 +154,24 @@
             </dependency>
             <dependency>
                 <groupId>de.jplag</groupId>
-                <artifactId>rlang</artifactId>
+                <artifactId>cpp</artifactId>
                 <version>${revision}</version>
             </dependency>
             <dependency>
                 <groupId>de.jplag</groupId>
-                <artifactId>cpp</artifactId>
+                <artifactId>golang</artifactId>
```

Comment:
have you sorted the modules alphabetically?

Reply:
Could be the pom sorting from spotless.

Reply:
Yes, I saw that the module list in the aggregator POM was now sorted, so I figured it would be neat if the dependencies would be sorted as well. Now it looks really stupid in the diff.